File: UPCODER/Cấu trúc dữ liệu/1. Tìm kiếm/TKTUANTU.py
def sequentialSearch(arr, n, x):
    for i in range(0, n):
        if arr[i] == x: 
            return i
    return -1

n, x = map(int, input().split())
arr = list(map(int, input().split()))

ans = sequentialSearch(arr, n, x)
print(ans)

File: UPCODER/Cấu trúc dữ liệu/1. Tìm kiếm/TKTUANTU.java
import java.util.Scanner;

class Main {
    public static int sequentialSearch(int[] arr, int n, int x) {
        for (int i = 0; i < n; i++)
            if (arr[i] == x) return i;
        return -1;
    }
     
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, x, ans;
        n = sc.nextInt();
        x = sc.nextInt();
        
        int arr[] = new int[n];
        for (int i = 0; i < n; i++)
            arr[i] = sc.nextInt();
        
        ans = sequentialSearch(arr, n, x);
        System.out.print(ans);
    }
}